scripts/reboot

```diff
@@ -24,6 +24,7 @@ fi
 REBOOT_USER=$(logname)
 PLATFORM=$(sonic-cfggen -H -v DEVICE_METADATA.localhost.platform)
 ASIC_TYPE=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
+SUBTYPE=$(sonic-cfggen -d -v DEVICE_METADATA.localhost.subtype)
```

Comment:
What if `subtype` is not defined? Would it throw?

Reply:
It doesn't throw, it just fails the check and reboot proceeds as usual

Reply:
interesting it doesn't throw if the lest key is not present, however it throw if intermediate ones are not present!
```
admin@STR43-0101-0101-01UT0:~$ sonic-cfggen -d -v DEVICE_METADATA.localhost1.subtype
Traceback (most recent call last):
  File "/usr/local/bin/sonic-cfggen", line 433, in <module>
    main()
  File "/usr/local/bin/sonic-cfggen", line 407, in main
    print(template.render(data))
  File "/usr/local/lib/python3.7/dist-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.7/dist-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/usr/local/lib/python3.7/dist-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 1, in top-level template code
  File "/usr/local/lib/python3.7/dist-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'localhost1'
admin@STR43-0101-0101-01UT0:~$ sonic-cfggen -d -v DEVICE_METADATA.localhost.subty
```